pom.xml

```diff
@@ -1479,16 +1479,6 @@
 
   <repositories>
     <repository>
-      <id>jboss-snapshots</id>
-      <url>https://repository.jboss.org/nexus/content/repositories/snapshots</url>
-      <snapshots>
-        <enabled>true</enabled>
-      </snapshots>
-      <releases>
-        <enabled>false</enabled>
-      </releases>
-    </repository>
-    <repository>
```

Comment:
Having it there it allows someone to build one module only, but that is probably not required.
It might be useful in case we depend on some other JBoss snapshots.
Yours case that the build started to download the dependencies was probably due to outdated local snapshots.
I would leave it in.